pkg/controller/tidb_control.go

```diff
@@ -41,7 +41,7 @@ type DBInfo struct {
 // TiDBControlInterface is the interface that knows how to manage tidb peers
 type TiDBControlInterface interface {
 	// GetHealth returns tidb's health info
-	GetHealth(tc *v1alpha1.TidbCluster) map[string]bool
+	GetHealth(tc *v1alpha1.TidbCluster, ordinal int32) (bool, error)
```

Comment:
tidb member controller will decide which member we need to request the health status